test/unit/core/plugins/json-schema-2020-12-samples/fn.js

```diff
@@ -2983,3 +2984,55 @@ describe("memoizedCreateXMLExample", () => {
     ).toEqual(updatedExpected)
   })
 })
+
+describe("mergeJsonSchema", function () {
```

Comment:
I wasn't sure how to test if we get to the correct function based on the spec, so only added the same test for both `json-schema-2020-12-samples` and `json-schema-5-samples` to check their behaviour.

Reply:
It would be hard to test actually. Let's assume that `wrapOAS31Fn` works as expected.